exporter/dynatraceexporter/metrics_exporter_test.go

```diff
@@ -483,3 +485,34 @@ func Test_exporter_start_InvalidHTTPClientSettings(t *testing.T) {
 		return
 	}
 }
+
+func Test_exporter_new_with_tags(t *testing.T) {
+	config := &config.Config{
+		Tags: []string{"test_tag=value"},
+	}
+
+	exp := newMetricsExporter(componenttest.NewNopExporterCreateSettings(), config)
+
+	assert.Equal(t, dimensions.NewNormalizedDimensionList(dimensions.NewDimension("test_tag", "value")), exp.defaultDimensions)
+}
+
+func Test_exporter_new_with_default_dimensions(t *testing.T) {
+	config := &config.Config{
+		DefaultDimensions: map[string]string{"test_dimension": "value"},
+	}
+
+	exp := newMetricsExporter(componenttest.NewNopExporterCreateSettings(), config)
+
+	assert.Equal(t, dimensions.NewNormalizedDimensionList(dimensions.NewDimension("test_dimension", "value")), exp.defaultDimensions)
+}
+
+func Test_exporter_new_with_default_dimensions_override_tag(t *testing.T) {
+	config := &config.Config{
+		Tags:              []string{"from=tag"},
+		DefaultDimensions: map[string]string{"from": "default_dimensions"},
+	}
```

Comment:
You could add a second value each in both and verify that you get three as a result.

Reply:
The equality check fails because order is not guaranteed. I can add this test but it will take some time.

Reply:
Alright, feel free to do it in a follow-up.